examples/datasource-basic/src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -17,19 +17,25 @@ export function ConfigEditor(props: EditorProps): ReactElement {
     <>
       <FieldSet label="General">
         <InlineField label="Time Field" tooltip="Default time field used when interpolate the $__timeFilter()">
-          <Input onChange={onTimeFieldChange} placeholder="time" value={jsonData?.defaultTimeField ?? ''} />
+          <Input
+            onChange={onTimeFieldChange}
+            placeholder="time"
+            aria-label="Macro default time field"
+            value={jsonData?.defaultTimeField ?? ''}
+          />
         </InlineField>
       </FieldSet>
 
       <FieldSet label="API Settings">
         <SecretFormField
           tooltip="API Key used to make calls to your data source"
-          isConfigured={Boolean(secureJsonData?.apiKey)}
+          isConfigured={Boolean(secureJsonFields['apiKey'])}
```

Comment:
**Nit:** I think we can use the dot notation here for accessing the property (I don't know if we have any standard for these)
```suggestion
          isConfigured={Boolean(secureJsonFields.apiKey)}
```